examples/spawn-and-move/abis/base/contracts/actions.json

```diff
@@ -235,7 +235,7 @@
       {
         "name": "player",
         "type": "core::starknet::contract_address::ContractAddress",
-        "kind": "data"
+        "kind": "key"
```

Comment:
any idea why does this changed?

Reply:
I guess something wasn't rebuilt during the PR that introduced this event (as I also have other diffs related to that).
Because it's a key: https://github.com/dojoengine/dojo/blob/810b0f45739d96f865fb6e402a30f4e3cf99a977/examples/spawn-and-move/src/actions.cairo#L33